### DISCUSSION TITLE
Success! Calibre 5.8.1, DeDRM 7.x, Digital editions 4.5.11, 2020 publication

### DISCUSSION THREAD

**Discussion by DaveEveritt** [apprenticeharper/DeDRM_tools#1460] [General]
First attempt ever at removing DRM. OS X 10.14.6.

Aim: remove DRM from a 2020 published book, downloaded and opened in (the awful and almost unreadable) Adobe Digital editions 4.5.11.

Steps taken:

1. Install the DeDRM plugin by choosing the downloaded zip file "DeDRM_plugin_7.0.3.zip" (or the [latest release](https://github.com/apprenticeharper/DeDRM_tools/releases)) which contains everything - the plugin now appears in the Calibre list of plugins on restart.
2. Delete and re-import my book into Calibre 5.8.1 from the Adobe "Digital Editions" folder on my computer.
3. Read the DeDRM guide, which mentioned also installing Obok, but Obok files are in a parent repo and have no zip file, so it's hard to see what to download. Decide to ignore Obok and continue.
4. Nothing in the Calibre interface or help docs explains how to go forward from here, so select my book and click "convert Books" in Calibre.
5. Choose EPUB for the format (top right) and hit 'okay' (bottom right).
6. Click the link to the resulting converted file in Calibre.
7. Drag the converted and DeDRM'd file into Apple Books and open it to check, it appears fine at first glance.
8. Open Apple Books for iOS on my phone, the newly-converted book imports automatically (I have sync Books across devices set up).

Success! Not all smooth sailing and much poring over an initially daunting and confusing workflow and a forest of version numbers (all of which proved irrelevant in the end) but a good outcome as soon as I'd found my way.

Feel free to share this for OS X Digital Editions 4.x and Calibre 5.x users.

**Comment by ElleKayEm** [comment]
Be aware that you can get hardened Adobe DRM which cannot be removed when using ADE 3 and later.  If you can download and read it with ADE 2.0.1 authorized with your Adobe ID, you know it has DRM that can be removed.

You only need Obok for books from Kobo Desktop or from a Kobo ereader.

If everything is working correctly, DRM removal happens when you add the book to calibre.  No need to convert unless you actually want the book in a different format.

**Reply by DaveEveritt** [reply]
some of this isn't at all obvious or easy to determine as a first-time user (e.g. that DRM is removed after DeDRM is installed at the very point the file is imported to Calibre) hence my "for dummies" version… and how would I know in advance that it has hardened DRM?

**Reply by ElleKayEm** [reply]
You won't know if an ebookstore is using hardened Adobe DRM in advance.  Hence the suggestion to use ADE 2.0.1.  If it won't download, you could perhaps ask the vendor for a refund.  Or at least know not to buy from them again.

**Reply by myastark** [reply]
> Install the DeDRM plugin by choosing the downloaded zip file "DeDRM_plugin_7.0.3.zip"

Choosing it where?

**Reply by ElleKayEm** [reply]
Latest pre-release here:  https://github.com/apprenticeharper/DeDRM_tools/releases/tag/7.0.3

**Reply by DaveEveritt** [reply]
Link to [all releases and pre-releases](https://github.com/apprenticeharper/DeDRM_tools/releases) added to my post - most recent is at the top of that page - probably best to get the most recent if you're using the latest reader software etc. ElleKayEm kindly added a direct link to the version I used.

**Comment by ElleKayEm** [comment]
Versions I use on Windows 10:
calibre 4.23 (64-bit)
DeDRM 6.8.1
Adobe Digital Editions 2.0.1
Kindle for PC 1.17

**Comment by jomandaCB** [comment]
Hi, I successfully use Calibre 4 on MacOS Mojave with the 6.8.1 DeDRM plugin on EPUB books from ADE 2.0.1
If I upgrade my Mac to Catalina or Big Sur, it will no longer run ADE 2 as it is a 32-bit app. Do the DeDRM Tools 7.1.0 work for books acquired with ADE 3 or 4, please?

**Reply by ElleKayEm** [reply]
It depends on whether the ebookstore is using hardened Adobe DRM or not.  I don't know of any way to find out for sure ahead of purchase.  I don't believe Kobo or Google are.  If they aren't, you'll be fine with Adobe 3 and higher.
